moss_web_demo_streamlit.py

```diff
@@ -58,7 +58,7 @@ def load_model():
 
 
 def generate_answer():
-   
+
```

Comment:
FYI, this file is indented with 3 spaces rather than 4.

Reply:
why is that? This is weird